### PR TITLE
Disable mission creating and searching

### PR DESCRIPTION
**Pull request by alastairdglennie**
As per upcoming miletsone, we only want to have a single mission (users can't create new missions). 

This flag temporarily disables the create mission button and seach box.

### CODE REVIEW

**Review by sebastianovide — COMMENTED** [review]
is LAUNCH_DAY in some how related to isMissionEnabled ?
(https://github.com/PlasticPatrol/PlasticPatrolWeb/blob/35bd95d1ff7d7d36a609fb2ba73631a846775d8d/src/custom/config.tsx#L22)

**Comment by alastairdglennie** [comment]
> is LAUNCH_DAY in some how related to isMissionEnabled ?
> (
> 
> https://github.com/PlasticPatrol/PlasticPatrolWeb/blob/35bd95d1ff7d7d36a609fb2ba73631a846775d8d/src/custom/config.tsx#L22
> 
> )

This refers to the test that @lizheard1 wants to run where there's a single mission and users can't create new ones. The easiest way of toggling this feature off seemed to be just adding a new flag like this

**Review by sebastianovide — APPROVED** [review]

File: src/pages/missions/MissionsHome.tsx [hunk 1]
```diff
@@ -133,8 +129,9 @@ export default function MissionsHome({}: Props) {
       label={"Missions"}
       navigationHandler={{ handleClose: handleClose }}
       className={classes.wrapper}
-      addAction={() => history.push(linkToCreateMission())}
+      addAction={config.LAUNCH_DAY_FEATURE_DISABLING ? undefined : () => history.push(linkToCreateMission())}
```
Comment (sebastianovide) [inline src/pages/missions/MissionsHome.tsx:132]: what about `!isMissionEnabled()` ? or `!isMissionLaunchDayEnabled()` and then updating this: https://github.com/PlasticPatrol/PlasticPatrolWeb/blob/d1862d79c5862696d5198bd85d397ae40c66241e/src/custom/featuresFlags.tsx#L8

**Review by sebastianovide — APPROVED** [review]

**Comment by sebastianovide** [comment]
@alastairdglennie I'm merging it assuming that you wanted to merge it... (if not, we can revert it)

**Pull request merged**